src/slack/message/index.js

```diff
@@ -41,6 +73,12 @@ const getLanguage = (record) => {
   return formattedLanguageList;
 };
 
+/**
+ * Format the task requester for display in slack
+ *
+ * @param {object} record - the requested task to get the requester from
+ * @returns {string} - formatted requester
```

Comment:
This may come up again, but any function that calls `getSection()` and returns its value is returning a section `object`.

You've already noted that in the JSDoc for `getSection()` itself:
https://github.com/astoria-tech/volunteer-dispatch/pull/50/files#diff-59061ffc59b16c2e4ace956ca59ebffbR10

Reply:
Actually, it looks like all of the other instances where you document this return value have it correctly described as an object 🌟

Reply:
👍 Good catch. Should be fixed now.